lib/http-proxy/passes/web-outgoing.js

```diff
@@ -80,12 +80,44 @@ var redirectRegex = /^30(1|2|7|8)$/;
    *
    * @api private
    */
-  function writeHeaders(req, res, proxyRes) {
+  function writeHeaders(req, res, proxyRes, options) {
+    var rewriteCookieDomainConfig = options ? options.cookieDomainRewrite : undefined;
```

Comment:
no need for ternary operator here. `var cookieDomainRewrite = options.cookieDomainRewrite`